src/adminjs-options.interface.ts

```diff
@@ -242,8 +242,17 @@ export interface AdminJSOptions {
    * rollup bundle options;
    */
   bundler?: BundlerOptions;
+
+  /**
+   * Additional settings.
+   */
+  settings?: Partial<AdminJsSettings>;
 }
 
+export type AdminJsSettings = {
```

Comment:
I would use `AdminJSSettings` to match vs `AdminJSOptions` (capitalized `JS`)

Reply:
Done 👍